starlark/src/environment/mod.rs

```diff
@@ -180,6 +197,23 @@ impl Environment {
     pub fn get_parent(&self) -> Option<Environment> {
         self.env.borrow().get_parent()
     }
+
+    pub fn with_set_constructor(&self, constructor: Box<Fn(Vec<Value>) -> ValueResult>) {
```

Comment:
This method is a public API method, maybe extensive comment would be good.

Reply:
Done